libfwupd/fwupd-client.c

```diff
@@ -106,14 +108,14 @@ G_DEFINE_AUTOPTR_CLEANUP_FUNC(CURLU, curl_url_cleanup)
 static void
 fwupd_client_curl_helper_free (FwupdCurlHelper *helper)
 {
+#ifdef HAVE_LIBCURL
 	if (helper->curl != NULL)
 		curl_easy_cleanup (helper->curl);
-#ifdef HAVE_LIBCURL_7_56_0
 	if (helper->mime != NULL)
 		curl_mime_free (helper->mime);
-#endif
 	if (helper->headers != NULL)
 		curl_slist_free_all (helper->headers);
+#endif
```

Comment:
it's a bit funny to have functions like fwupd_client_curl_helper_free when you don't actually have curl support..

Reply:
Totally valid, fixed.